cmd/acr/purge.go

```diff
@@ -28,6 +29,9 @@ const (
 	purgeExampleMessage    = `  - Delete all tags that are older than 1 day in the example.azurecr.io registry inside the hello-world repository
     	acr purge -r example --filter "hello-world:.*" --ago 1d
 
+  - Delete all tags that are older than 7 days in the example.azurecr.io registry inside all repositories
+	    acr purge -r example --filter ".*:.*" --ago 7d 
```

Comment:
Can we also update the documentation in README? 🙂

Reply:
Done, please check